src/util/is_char_in_unicode_block.js

```diff
@@ -244,6 +248,8 @@ const unicodeBlockLookup: UnicodeBlockLookup = {
     // 'Tangut': (char) => char >= 0x17000 && char <= 0x187FF,
     // 'Tangut Components': (char) => char >= 0x18800 && char <= 0x18AFF,
     // 'Kana Supplement': (char) => char >= 0x1B000 && char <= 0x1B0FF,
+    // 'Kana Extended-A': (char) => char >= 0x1B100 && char <= 0x1B12F,
+    // 'Nushu': (char) => char >= 0x1B170 && char <= 0x1B2FF,
```

Comment:
Nushu and CJK Unified Ideographs Extension F consist of characters that should be given ideographic breaking and vertical layout, but for our lack of astral plane support: #4001.